extensions/optaplanner/runtime/src/main/java/io/quarkus/optaplanner/nativeimage/KogitoSubstitutions.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.optaplanner.nativeimage;
+
+import java.lang.reflect.Method;
+
+import com.oracle.svm.core.annotate.Alias;
+import com.oracle.svm.core.annotate.RecomputeFieldValue;
+import com.oracle.svm.core.annotate.TargetClass;
+
+@TargetClass(className = "org.drools.core.rule.builder.dialect.asm.ClassGenerator")
+final class Target_org_drools_core_rule_builder_dialect_asm_ClassGenerator {
```

Comment:
-1 
This is a substitution that should happen in Kogito (= Drools), not in OptaPlanner. It's not OptaPlanner alone that will suffer from it.  OptaPlanner should not work around it, Kogito should fix it. Kogito 0.8.0 is borked, we need a Kogito 0.8.1 release.

Reply:
If we want to get the PR in, there is no other way. If you folks want to wait for a Kogito release that doesn't suffer from this, that's up to you :)

Reply:
I am willing to comprise to merge it like this for Quarkus 1.3.0 as a temporarily workaround (*), but there's a commitment to revert it by Quarkus 1.3.1 by releasing and using Kogito 0.8.1. @mariofusco wdyt?

(*) Let's add a comment stating that it's a temporarily workaround.

Reply:
In any case, this PR isn't making it into any 1.3.x line. We are talking purely for 1.4, whether it goes in as is, or with a new Kogito version